docs/.vitepress/theme/components/Feedback/Feedback.vue

```diff
@@ -35,6 +43,11 @@ function encode(data: Record<string, string>) {
 // We send a form to Netlify: https://docs.netlify.com/forms/setup/
 function submitForm() {
   isSending.value = true;
+  sendEvent({
```

Comment:
Moved the `gtag` event to when a user submits the form, instead of when a thumbs up/down is selected.

There is now an option to "go back" or "cancel" filling out the form, so it would probably be overkill to send events on the selection buttons.